README.md

```diff
@@ -324,3 +375,63 @@ the name.
 Refer to the [Building Web
 Driver](https://github.com/SeleniumHQ/selenium/wiki/Building-WebDriver)
 wiki page for the last word on building the bits and pieces of Selenium.
+
+## Bazel Installation/Troubleshooting
+
+### MacOS
+
+Bazel for Mac requires some additional steps to configure it properly. First things first: use
+the Bazelisk project (courtesy of philwo), a pure golang implementation of Bazel. In order to 
+install Bazelisk, first verify that your XCode will cooperate: execute the following command:
+
+`xcode-select -p`
+
+If the value is `/Applications/Xcode.app/Contents/Developer/`, you can proceed with bazelisk
+installation. If, however, the return value is `/Library/Developer/CommandLineTools/`, you'll
+need to redirect the XCode system to the correct value. 
+
+```
+sudo xcode-select -s /Applications/Xcode.app/Contents/Developer/
+sudo xcodebuild -license
+```
+
+The first command will prompt you for a password. The second step requires you to read a new XCode 
+license, and then accept it by typing "agree":
+
+If you've already installed bazel, it will need to reset its config data, which will redirect internal 
+commands to the correct XCode system. If you haven't installed bazel yet, you may skip this step:
+ 
+```
+bazel clean --expunge
+```
+
+(Thanks to [this thread](https://github.com/bazelbuild/bazel/issues/4314) for these steps)
+
+
+Once that's done, it's time to install bazelisk. 
+
+First, install golang (if you haven't already):
+
+```brew install go```
+
+Once that's done (verify with `which go`), use go's installer to get bazelisk.
+
+```go get github.com/bazelbuild/bazelisk```
+
+This will download and install bazelisk. To use the `bazel` command as we do everywhere in this 
+documentation, you'll need to create a symlink:
+
+```
+ln -s /Users/<username>/go/bin/bazelisk /usr/local/bin/bazel
+```
+
+### Coursier
+
+Coursier is a dependency management tool used by Selenium to fetch the latest
```

Comment:
Coursier is just really nice to have. We don't use it at all in the build or the project in general.